geoh5py/ui_json/utils.py

```diff
@@ -39,11 +39,19 @@ def flatten(ui_json: dict[str, dict]) -> dict[str, Any]:
                 field = "value" if truth(ui_json, name, "isValue") else "property"
                 if not truth(ui_json, name, "enabled"):
                     data[name] = None
+                elif (
+                    "groupValue" in value
+                    and "value" in value
+                    and value["groupValue"] is not None
+                ):
+                    data[name] = {
+                        "groupValue": value["groupValue"],
```

Comment:
Same here

Reply:
or it's in flatten, 
or it's in promote.

I chose promote as I feel it makes more sense, but it's as you want

Reply:
Yep, agreed.